sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/Field.java

```diff
@@ -224,6 +224,17 @@
     @JsonProperty(value = "fields")
     private List<Field> fields;
 
+    /*
+     * A value indicating whether the field will be returned in a search
+     * result. This property must be true for key fields, and must be null for
+     * complex fields. You can hide a field from search results if you want to
+     * use it only as a filter, for sorting, or for scoring. This property can
+     * also be changed on existing fields and enabling it does not cause an
+     * increase in index storage requirements.
+     */
+    @JsonProperty(value = "hidden")
```

Comment:
This may cause issues if the deserializer on the service side doesn't ignore unknown properties. Is there a way this change could be done which doesn't include adding a new property to the model?

Reply:
Can use @JsonIgnore